python/lsst/ts/tcpip/base_one_client_read_loop_server_test_case.py

```diff
@@ -0,0 +1,76 @@
+# This file is part of ts_tcpip.
+#
+# Developed for the Rubin Observatory Telescope and Site System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <https://www.gnu.org/licenses/>.
+
+import abc
+import asyncio
+import contextlib
+import logging
+import typing
+import unittest
+
+from .client import Client
+from .one_client_read_loop_server import OneClientReadLoopServer
+
+__all__ = ["BaseOneClientReadLoopServerTestCase"]
+
+# Standard timeout in seconds.
+TIMEOUT = 60
```

Comment:
This is only used to wait for the server to notice that the client has connected. As such, you might consider making it much shorter and giving it a more evocative name. Even 10 seconds is crazy generous.

(60 seconds in salobj is because DDS can be so slow)